dashboards/nvidia-dcgm/nvidia-dcgm.json

```diff
@@ -0,0 +1,246 @@
+{
+  "name": "NVIDIA",
+  "description": null,
+  "pages": [
+    {
+      "name": "Overview",
+      "description": null,
+      "widgets": [
+        {
+          "title": "",
+          "layout": {
+            "column": 1,
+            "row": 1,
+            "width": 2,
+            "height": 2
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.markdown"
+          },
+          "rawConfiguration": {
+            "text": "![NVIDIA DCGM](https://assets.nvidiagrid.net/ngc/logos/DCGM.png)"
```

Comment:
Use the NVIDIA logo from - https://github.com/newrelic/newrelic-quickstarts/assets/104448291/461421da-3f8b-4d71-bac7-2e20d58b4180 Check this in dark theme too.